dogs/src/App.js

```diff
@@ -31,11 +31,9 @@ class App extends React.Component {
   };
```

Comment:
Great job on the sprint challenge Aaron. 
- You successfully completed the MVP by fetching data from 3rd party API and displayed it on the webpage. You have properly split functionality into smaller components and passed necessary data as props.
- You have also gone above and beyond and completed a stretch goal and fetch the Dogs API. Awesome job on that ! Not just that but building pagination system you are able to load and display next or previous set of data on Your app without refreshing the page based on React convention.

- Your project is styled properly and design looks good. It is fully responsive and looks great on both mobile and tablet views so You certainly did great job with this one.

- Your code is clean and Your git workflow is fluid.
Overall:
  * Able to fetch and display data (MVP)
  * Good answers
  * Possible to switch to next “page” by loading next set of data (stretch)
  * Clean and modular code with multiple components
  * Implemented styling
  * Responsive design
  * Fluid git workflow

Possible Improvements:
* Add some kind of loading icon or information that data is being fetched. After loading Your app it takes some time to load and render everything so it would be great fit here